components/script/dom/htmlformelement.rs

```diff
@@ -302,17 +300,18 @@ impl HTMLFormElement {
 
             // Step 5
             // https://tools.ietf.org/html/rfc7578#section-4
-            result.push_str(&*format!("\r\n--{}\r\n", boundary));
+            let mut boundary_bytes = format!("--{}\r\n", boundary).into_bytes();
```

Comment:
Like this https://dxr.mozilla.org/mozilla-central/source/dom/html/HTMLFormSubmission.cpp#616? ` mPostDataChunk` is global to the class but I think it does start with a `--` followed by boundary when processing a multipart-formdata.